src/main/java/ecdar/model_canvas/arrow_heads/SimpleArrowHead.java

```diff
@@ -1,15 +1,14 @@
 package ecdar.model_canvas.arrow_heads;
 
-import ecdar.presentations.Grid;
 import ecdar.utility.colors.Color;
 import ecdar.utility.Highlightable;
 import ecdar.utility.helpers.SelectHelper;
 import javafx.scene.shape.Line;
 
 public class SimpleArrowHead extends ArrowHead implements SelectHelper.Selectable, Highlightable {
 
-    private static final double TRIANGLE_LENGTH = Grid.GRID_SIZE * 1.5;
-    private static final double TRIANGLE_WIDTH = Grid.GRID_SIZE;
+    private static final double TRIANGLE_LENGTH = 15;
+    private static final double TRIANGLE_WIDTH = 10;
```

Comment:
Consider using `Ecdar.CANVAS_PADDING`

Reply:
Changing the padding would change the size of the arrow head, which IMO is unexpected

Reply:
Changing the padding would change the size of the arrow head, which IMO is unexpected